duva-client/src/observability.rs

```diff
@@ -0,0 +1,86 @@
+use opentelemetry::KeyValue;
+use opentelemetry_appender_tracing::layer::OpenTelemetryTracingBridge;
+use opentelemetry_otlp::WithExportConfig;
+use opentelemetry_sdk::Resource;
+use opentelemetry_sdk::logs::SdkLoggerProvider;
+use std::sync::LazyLock;
+use std::time::Duration;
+use tracing_subscriber::{EnvFilter, Layer, layer::SubscriberExt, util::SubscriberInitExt};
+use uuid::Uuid;
+
+pub struct ObservabilityConfig {
+    pub log_level: String,
+}
+
+impl Default for ObservabilityConfig {
+    fn default() -> Self {
+        Self { log_level: "info".to_string() }
+    }
+}
+
+impl ObservabilityConfig {
+    pub fn new() -> Self {
```

Comment:
remove this line if it doesn't accept any param ;)

Reply:
True! Will remove ;)